config/locales/views/advice_pages/baseload.yml

```diff
@@ -13,7 +13,7 @@ en:
       analysis_baseload_low_details_html: Your school's baseload is low, which is good and as a result you are saving %{estimated_savings} per year versus the average of other schools of a similar size.
       analysis_electricity_baseload_chart_explanation: Each point on this chart represents the average over the hours a school is closed on that day. You can click on a point in the chart and it will drill down to show you the usage on that day. Ideally, there should be little variation between different days and seasons.
       analysis_electricity_baseload_chart_subtitle: This chart shows the electricity baseload for your school between %{start_month_year} and %{end_month_year}.
-      analysis_electricity_baseload_chart_title: Electricity baseload for the last 12 months
+      analysis_electricity_baseload_chart_title: Electricity baseload for the last %{months_analysed} months
```

Comment:
does this need a one and other key? (e.g. for 1 "Electricity baseload for the last month") (similarly the 2 translations below)

Reply:
I'll handle this in the next PR